cpp/src/io/csv/reader_impl.cu

```diff
@@ -377,8 +377,7 @@ table_with_metadata reader::impl::read(size_t range_offset,
         out_columns.emplace_back(
           cudf::strings::replace(col->view(), dblquotechar, quotechar, -1, mr_));
       } else {
-        out_columns.emplace_back(
-          make_column(column_types[i], num_records, out_buffers[i], stream, mr_));
+        out_columns.emplace_back(make_column(out_buffers[i], stream, mr_));
```

Comment:
Please forgive this tangent, but in this context, there is no advantage to using `emplace_back()` instead of `push_back()`, is there? (I mean, apart from consistency with the rest of the code.)

Reply:
yeah, AFAIK it's a move ctor either way